readme.md

```diff
@@ -35,35 +33,47 @@ Then go to http://localhost:8080/ in your browser.
 
 The project depends on the following packages in order to run properly.
 
-```JSON
-{
-	"dependencies": {
-		"bootstrap": "^4.1.3",
-		"date-fns": "^1.29.0",
-		"highlightjs": "^9.10.0",
-		"jquery": "^3.3.1",
-		"katex": "^0.10.0-rc.1",
-		"mithril": "^1.1.6",
-		"popper.js": "^1.14.4",
-		"quill": "^1.3.6"
-	},
-	"devDependencies": {
-		"@types/jquery": "^3.3.6",
-		"@types/mithril": "^1.1.12",
-		"@types/node": "^10.9.4",
-		"@types/quill": "^2.0.0",
-		"css-loader": "^1.0.0",
-		"node-sass": "^4.9.3",
-		"sass-loader": "^7.1.0",
-		"style-loader": "^0.23.0",
-		"ts-loader": "^5.1.0",
-		"tslint": "^5.11.0",
-		"tslint-react": "^3.6.0",
-		"typescript": "^3.0.3",
-		"url-loader": "^1.1.1",
-		"webpack": "^4.18.0",
-		"webpack-cli": "^3.1.0",
-		"webpack-dev-server": "^3.1.8"
-	}
-}
+Mithril is a modern client-side Javascript framework for building Single Page Applications.
+It's small (< 8kb gzip), fast and provides routing and XHR utilities out of the box.
+For the mitrhil docs click [here](https://mithril.js.org/api.html)
+```
+"mithril": "^1.1.6"
+```
+
+Bootstrap is a sleek, intuitive, and powerful front-end framework for faster and easier web development. 
+For the bootstrap docs click [here](https://getbootstrap.com/)
+```
+"bootstrap": "^4.1.3"
+```
+
+Date-fns is a zero-dependency module that is able to manipulate JavaScript dates.
+```
+"date-fns": "^1.29.0"
+```
+
+Syntax highlighting for the Web.
+```
+"highlightjs": "^9.10.0"
+```
+
+JavaScript library for DOM operations.
+```
+"jquery": "^3.3.1"
+```
+
+KaTeX is a fast, easy-to-use JavaScript library for TeX math rendering on the web.
+```
+"katex": "^0.10.0-rc.1"
+```
+
+A popper is an element on the screen which "pops out" from the natural flow of your application.
+Common examples of poppers are tooltips, popovers and drop-downs.
+```
+"popper.js": "^1.14.4"
+```
+
+QuillJS is a modern rich text editor built for compatibility and extensibility.
```

Comment:
We don't use this package anymore, we use https://marked.js.org/#/README.md instead.